Evershop.Tests/Evershop.Tests.API/Models/LoginResponseData.cs

```diff
@@ -11,6 +11,12 @@ internal class BearerId
         {
             [JsonProperty("sid")]
             public string Sid { get; set; }
+
+            [JsonProperty("uuid")]
+            public string Uuid { get; set; }
```

Comment:
these properties can't be part of LoginResponseData. This class represents body structure of Login response. CreateProduct response has another body, so it must be another class which will be used to parse response body into the instance of CreateProductResponseData (for example)